interp/src/interpreter/steppers/component_interpreter.rs

```diff
@@ -367,7 +367,7 @@ impl<'a, 'outer> Primitive for ComponentInterpreter<'a, 'outer> {
         Some(Box::new(self.get_env()))
     }
 
-    fn serialize(&self) -> crate::primitives::Serializeable {
+    fn serialize(&self, _signed: bool) -> crate::primitives::Serializeable {
```

Comment:
Comment on why this is unused here?

Reply:
It doesn't make sense for a component itself to be marked as signed for serialization, but the trait still needs the argument to be there